Sources/Workspace/Destination.swift

```diff
@@ -111,35 +111,41 @@ public struct Destination: Encodable, Equatable {
             }
             sdkPath = AbsolutePath(sdkPathStr)
         }
+#else
+        sdkPath = nil
```

Comment:
This will also need a follow up.  The Windows setup that I have actually does the split SDK setup and uses the `-sdk` flag for Windows.

Reply:
For now I've adjusted the build script on my machine to install the SDK inside the toolchain, mostly to keep the directory layout consistent with Linux toolchains. It works well for me.
But I agree it would be nice to have the support for Xcode-like layout on non-macOS with SDKs placed outside of the xctoolchain directory.